sdk/purview/purview-administration-rest/.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "plugins": ["@azure/azure-sdk"],
+  "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"],
+  "rules": {
+    "@azure/azure-sdk/ts-modules-only-named": "warn",
+    "@azure/azure-sdk/ts-apiextractor-json-types": "warn",
+    "@azure/azure-sdk/ts-package-json-types": "warn",
+    "@azure/azure-sdk/ts-no-namespaces": "warn",
+    "@azure/azure-sdk/ts-package-json-engine-is-present": "warn",
+    "tsdoc/syntax": "warn"
```

Comment:
I think code gen should generate code that respects most of these rules. Is that tracked in an issue?

Reply:
Yup I think we can remove most of them. There is one that we need to keep for RLCs  `@azure/azure-sdk/ts-modules-only-named` to allow having a default export.

For this particular package we can remove the exception, since we are just doing named exports due to the multi-client nature of the package